### PR TITLE
docstring only formatting changes: quantize.py,  fake_quantize.py, observer.py

### PR DESCRIPTION
**Pull request by gottbrath**

### CODE REVIEW

**Review by z-a-f — APPROVED** [review]
LGTM

**Comment by gottbrath** [comment]
This fixes https://github.com/pytorch/pytorch/issues/26834, and helps fix https://github.com/pytorch/pytorch/issues/27428, and https://github.com/pytorch/pytorch/issues/26470, and https://github.com/pytorch/pytorch/issues/26469

File: torch/quantization/fake_quantize.py [hunk 1]
```diff
@@ -40,13 +40,6 @@ class FakeQuantize(Module):
...
                            provides a method to calculate scale and zero-point.
 
-    """
-    Args:
-        `observer`: Observer module that records stats of input tensor
-        `quant_min`: Tensors are fake-quantized corresponding to the
-        `quant_max`: A function that calculates quantization parameters
-        given the stats
-        `observer_kwargs`
     '''
```
[the remark this answers is not in the document]
Comment (gottbrath) [reply torch/quantization/fake_quantize.py]: ok .. will do.

**Comment by gottbrath** [comment]
I see a notice that merging is blocked but I also see a green button that says "import to phabricator" Will try clicking.

**Comment by gottbrath** [comment]
based on Soumith's comments on https://github.com/pytorch/pytorch/pull/27414 there might be some merge conflicts here. Will see if I can diagnose and resolve.

**Comment by gottbrath** [comment]
I think that I have revised the PR i sent Soumith for v1.3.0 so that it won't conflict with this.

**Pull request closed without merging**